fluffd/fluffd.js

```diff
@@ -20,7 +20,7 @@
 
 // System: HTTP server and Bluetooth Low Energy Library
 const winston = require("./logger");
-const noble = require("noble");
+const noble = require("@abandonware/noble");
```

Comment:
Thanks for this PR, it helped me a lot to make it work on a RPi 4.
However, it should be noted that package.json must also be updated:

```
	"dependencies": {
		"@abandonware/noble": "1.9.2-10",
```